core/dyn-configs/src/lib.rs

```diff
@@ -0,0 +1,16 @@
+#![doc = include_str!("../README.md")]
+
+use once_cell::sync::Lazy;
+use std::sync::atomic::{AtomicU64, Ordering};
+
+// NOTE: AtomicU64 is the same unit as BlockHeight, and use to store the expected blockheight to
+// shutdown
+pub static EXPECTED_SHUTDOWN_AT: Lazy<AtomicU64> = Lazy::new(|| AtomicU64::new(0));
```

Comment:
```suggestion
pub static EXPECTED_SHUTDOWN_AT: AtomicU64 = AtomicU64::new(0);
```

drive by, but this doesn't need once_cell

Reply:
Thank you. :smile: